src/Index.ts

```diff
@@ -253,3 +253,18 @@ export {SlideY} from './animations/SlideY';
 

```

Comment:
This file should be deleted, the file Index.ts on the root automatically export everything

Reply:
Nice, it'll be my pleasure to kill that file.